### ISSUE TITLE
Any SSR version ?

### ISSUE THREAD

**Issue by usb248** [gijsroge/vue-responsive-menu#4]
In order keep navigation menu in the source of page et able to crawle menu for SEO purpose

**Comment by gijsroge** [comment]
@usb248 sorry for the late response but i'm not sure what you mean.

If you use nuxt you can just put it between a `<client-only></client-only>` tag and it will not SSR 2 different menus.

**Comment by usb248** [comment]
@gijsroge If put my menu client side, there is no benefit with SEO and therefore, it's a very bad practice.

**Comment by gijsroge** [comment]
@usb248 true, so you would like to just SSR the entire menu, and only apply the responsive navigation logic on the client..

I'll be open for anyone to submit a pull request. This shouldn't be too difficult.
